sdk/core/azure-core/src/io/body_stream.cpp

```diff
@@ -75,10 +75,11 @@ std::vector<uint8_t> BodyStream::ReadToEnd(Context const& context)
   }
 }
 
-int64_t MemoryBodyStream::OnRead(uint8_t* buffer, int64_t count, Context const& context)
+int64_t MemoryBodyStream::OnRead(uint8_t* buffer, size_t count, Context const& context)
```

Comment:
On 32-bit systems this puts an upper limit of 4GB on the buffer size and the file size.  That means our API isn't compatible with 32-bit OS using LFS.